packages/swapper/src/swappers/thorchain/utils/constants.ts

```diff
@@ -1,2 +1,7 @@
 export const MAX_THORCHAIN_TRADE = '100000000000000000000000000'
 export const MAX_ALLOWANCE = '100000000000000000000000000'
+export const THOR_MINIMUM_PADDING = 1.2
+export const THOR_ETH_GAS_LIMIT = '100000' // for sends of eth / erc20 into thorchain router
+export const THORCHAIN_FIXED_PRECISION = 8 // limit values are precision 8 regardless of the chain
+export const THOR_BTC_SIZE = 1000 // used for estimating thorchain side fees
+export const THOR_ETH_GAS = 120000 // used for estimating thorchain side fees
```

Comment:
thanks for pulling this out - we have two very similar values here, i understand they're for different things, can we rename this one to be slightly more specific here, or add an additional comment?